cmd/webhook/main.go

```diff
@@ -90,7 +90,7 @@ func realMain(ctx context.Context) error {
 	shutdownCtx, done := context.WithTimeout(context.Background(), 5*time.Second)
 	defer done()
 
-	if err := webhookServer.Cleanup(); err != nil {
```

Comment:
If you make this `Close`, then you satisfy the `io.Closer` interface, which could be helpful.